osmchadjango/users/utils.py

```diff
@@ -25,9 +22,9 @@ def update_user_name(user):
     """
     try:
         uid = user.social_auth.get(provider='openstreetmap-oauth2').uid
-        url = '{}/api/0.6/user/{}/'.format(settings.OSM_SERVER_URL, uid)
-        data = ET.fromstring(requests.get(url).content)
-        display_name = data.find('user').get('display_name')
+        url = f'{settings.OSM_SERVER_URL}/api/0.6/user/{uid}.json'
+        data = requests.get(url, headers=settings.OSM_API_USER_AGENT).json()
+        display_name = data['user']['display_name']
```

Comment:
Looks good; is switching from XML to JSON API endpoints something we want to do systematically across the whole codebase in the future?

Reply:
Yes, I think we mostly use the JSON already.